libcloudforensics/providers/aws/forensics.py

```diff
@@ -166,16 +167,22 @@ def StartAnalysisVm(vm_name,
     default_availability_zone (str): Default zone within the region to create
         new resources in.
     boot_volume_size (int): The size of the analysis VM boot volume (in GB).
-    cpu_cores (int): Optional. The number of CPU cores to create the machine
-        with. Default is 4.
     ami (str): Optional. The Amazon Machine Image ID to use to create the VM.
         Default is a version of Ubuntu 18.04.
+    cpu_cores (int): Optional. The number of CPU cores to create the machine
+        with. Default is 4.
     attach_volume (AWSVolume): Optional. The volume to attach.
     device_name (str): Optional. The name of the device (e.g. /dev/sdf) for
         the volume to be attached. Mandatory if attach_volume is provided.
     dst_account (str): Optional. The AWS account in which to create the
         analysis VM. This is the profile name that is defined in your AWS
         credentials file.
+    ssh_key_name (str): Optional. A SSH key pair name linked to the AWS
+        account to associate with the VM. If none provided, the VM can only
+        be accessed through in-browser SSH from the AWS management console
+        with the EC2 client connection package. Note that if this package
+        fails to install on the target VM, then the VM will not be accessible.
+        It is therefore recommended to fill in this parameter.
```

Comment:
I'm not sure about the docstring here. "SSH key pair"? Don't you only need the public key? "This package fails to install" what package are we talking about? Same goes for the other docstring

Reply:
I wanted to be consistant with https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.run_instances

```
KeyName (string) --
The name of the key pair.
```

For the package, rephrased.